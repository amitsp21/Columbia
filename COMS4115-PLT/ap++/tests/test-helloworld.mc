/*
 * "Hello World" - Literal Print Test
 */

int main()
{
  printi(1234);
  printf(123.4);
  printb(true);
  prints("Hello World!");
  return 0;
}
