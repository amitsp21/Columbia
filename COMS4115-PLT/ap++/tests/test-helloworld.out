1234
123.4
1
Hello World!